Code/Framework/AzToolsFramework/AzToolsFramework/ComponentMode/ComponentModeCollection.cpp

```diff
@@ -461,7 +473,7 @@ namespace AzToolsFramework
 
                 if (m_selectedComponentModeIndex < m_activeComponentTypes.size())
                 {
-                    // notify other systems ComponentMode actions have changed
+                    // notify other systems ComponentMode has changed
```

Comment:
nit: I think here we could say `// notify other systems the active ComponentMode has changed` to be more precise (in the case we can have more than one enabled but not active at a time such as the Spline and Tube Shape)